main.go

```diff
@@ -319,6 +344,24 @@ func main() {
 			os.Exit(1)
 		}
 		protoList.Entry = append(protoList.Entry, site)
+
+		// Generate plaintext list
+		if *exportLists != "" {
+			if existList != nil {
+				generatePlainTextList(existList, refName, pl)
+			} else {
+				exportedListSlice := strings.Split(*exportLists, ",")
+				for _, exportedListName := range exportedListSlice {
+					fileName := filepath.Join(dir, exportedListName)
+					_, err := os.Stat(fileName)
+					if err == nil || os.IsExist(err) {
+						existList = append(existList, exportedListName)
+					} else {
+						fmt.Printf("'%s' list does not exist in '%s' directory.\n", exportedListName, dir)
+					}
+				}
+			}
+		}
```

Comment:
If happens that when ranging, the first `refName` is the one you want to export, then it cannot be exported.
It should be finished before ranging.